src/System.IO.FileSystem/tests/Directory/GetFileSystemEntries_str_str.cs

```diff
@@ -639,6 +639,13 @@ public void SearchPatternLongPath()
 
             if (TestFiles)
             {
+                if (PlatformDetection.IsNetNative)
+                {
+                    // UapAot throws if the Directory of the file to create doesn't exist.
+                    Assert.Throws<DirectoryNotFoundException>(() => File.Create(longFullname));
```

Comment:
> why is this correct?

In this case it shouldn't be. We should be able to create this file. Something might be wrong with \\?\ insertion for File.Create().